docs/installation/linux/fedora.md

```diff
@@ -12,80 +12,96 @@ weight=-3
 
 # Fedora
 
-Docker is supported on Fedora version 22, 23, and 24. This page instructs you to install
-using Docker-managed release packages and installation mechanisms. Using these
-packages ensures you get the latest release of Docker. If you wish to install
-using Fedora-managed packages, consult your Fedora release documentation for
-information on Fedora's Docker support.
+Docker is supported on Fedora version 22, 23, and 24. This page instructs you to
+install using Docker-managed release packages and installation mechanisms. Using
+these packages ensures you get the latest release of Docker. If you wish to
+install using Fedora-managed packages, consult your Fedora release documentation
+for information on Fedora's Docker support.
```

Comment:
s/Fedora's Docker support/Docker support in Fedora

I try to avoid personifying computery things by giving them ownership if I can avoid it with a rewrite.